projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1347,7 +1347,7 @@ private boolean getContainer(List<String> options, List<String> parameters) {
     String containerName = parameters.get(0);
     Container container = _workHelper.getContainer(containerName);
     if (container != null) {
-      _logger.output(container.getTestrigs() + "\n");
+      _logger.output(container + "\n");
```

Comment:
Should I modify the code here to get the same output? In my point of view, getContainer should return information about a container, rather than the testrig names in the container (which sounds like a listTestrig call). 
The current output look like:
```
Client output:
Container{name=containerName, createdAt=2017-09-12 14:58:53, testrigsCount=0, analysesCount=0}

use browser:
{
"name": "containerName",
"createdAt": "2017-09-12 14:58:53",
"testrigsCount": 0,
"analysesCount": 0
}
```